test/e2e/models/CodeServer.ts

```diff
@@ -296,6 +322,21 @@ export class CodeServerPage {
     return visible
   }
 
+  /**
+   * Checks if the test extension loaded
+   */
+  async isTestExtensionLoaded() {
+    const selector = "text=test extension loaded"
+    this.codeServer.logger.debug("Waiting for test extension to load...")
+
+    await this.page.waitForLoadState("load")
```

Comment:
We already `await` the load event on every page load (via `reloadUntilEditorIsReady`) so we can probably omit.

Reply:
Good call! I'll remove this.